src/main/java/org/apache/commons/imaging/formats/webp/chunks/WebPChunk.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.webp.chunks;
+
+import org.apache.commons.imaging.ImagingException;
+import org.apache.commons.imaging.common.BinaryFileParser;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * A WebP image is composed of several chunks. This is the base class for the chunks,
+ * used by the parser.
+ *
+ * @see <a href="https://developers.google.com/speed/webp/docs/riff_container">WebP Container Specification</a>
```

Comment:
Maybe this should be documented in the parser as well.

Reply:
None of the parsers for the other formats have javadoc, so I'm confused as to where to put the documentation.

Reply:
If the others are not doing that, then no worries @Glabo :) I spoke without looking at the other parsers (more a note as I did a first pass without the IDE). Thanks for replying it!